docs/src/usage.txt

```diff
@@ -1,30 +1,26 @@
-tealdeer 1.6.1
-Danilo Bargen <mail@dbrgn.ch>, Niklas Mohrin <dev@niklasmohrin.de>
-A fast TLDR client
+tealdeer 1.2.3: A fast TLDR client
 
-USAGE:
-    tldr [OPTIONS] [COMMAND]...
+Usage: tldr [OPTIONS] [COMMAND]...
 
-ARGS:
-    <COMMAND>...    The command to show (e.g. `tar` or `git log`)
+Arguments:
+  [COMMAND]...  The command to show (e.g. `tar` or `git log`)
 
-OPTIONS:
-    -l, --list                    List all commands in the cache
-    -f, --render <FILE>           Render a specific markdown file
-    -p, --platform <PLATFORMS>    Override the operating system [possible values: linux, macos,
-                                  windows, sunos, osx, android]
-    -L, --language <LANGUAGE>     Override the language
-    -u, --update                  Update the local cache
-        --no-auto-update          If auto update is configured, disable it for this run
-    -c, --clear-cache             Clear the local cache
-        --pager                   Use a pager to page output
-    -r, --raw                     Display the raw markdown instead of rendering it
-    -q, --quiet                   Suppress informational messages
-        --show-paths              Show file and directory paths used by tealdeer
-        --seed-config             Create a basic config
-        --color <WHEN>            Control whether to use color [possible values: always, auto,
-                                  never]
-    -v, --version                 Print the version
-    -h, --help                    Print help information
+Options:
+  -l, --list                  List all commands in the cache
+  -f, --render <FILE>         Render a specific markdown file
+  -p, --platform <PLATFORMS>  Override the operating system [possible values: linux, macos, windows,
+                              sunos, android]
```

Comment:
The help now doesn't list aliases anymore. I changed the code to use "macos" as main name and "osx" as alias (not vice versa), so that "macos" is shown in the help by default. "osx" still works though.

Reply:
For it to show aliases, you now have to use `visible_alias`, if I recall correctly. You can have a mix of visible and invisible aliases.

Reply:
It seems that `visible_alias` is only available on commands and args, but not on `PossibleValue`.